delete.go

```diff
@@ -113,11 +113,9 @@ func delete(containerID string, force bool) error {
 	return removeCgroupsPath(cgroupsPathList)
 }
 
-func deletePod(podID string, forceStop bool) error {
-	if forceStop {
-		if _, err := vc.StopPod(podID); err != nil {
-			return err
-		}
+func deletePod(podID string) error {
```

Comment:
No we should not be able to delete a running container, and this is checked by the code here https://github.com/clearcontainers/runtime/pull/383/files/618ab6482064bc977f802abed9084514e5569d45..4dc190d917a7eb21b00db85a0e0c0cf746979a1e#diff-635f3cde55281671a8db5283d1597a71R84